gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -303,6 +303,7 @@ public GsonBuilder newBuilder() {
    * @deprecated This method by accident exposes an internal Gson class; it might be removed in a
    * future version.
    */
+  @SuppressWarnings("exports")
```

Comment:
Eclipse showed a warning here because `Excluder` is not exported by the module descriptor (which is why we deprecated this method).
`"exports"` seems to be an Eclipse specific warning name.